src/method_ilan.jl

```diff
@@ -76,6 +76,8 @@ function ilan(
     check_error_every=30,
     inner_solver_method=DefaultInnerSolver(),
     Compute_Bmul_method::Type{T_y0}=Compute_Bmul_method_Auto,
```

Comment:
I think the kwarg `Compute_Bmul_method` should start with a small letter. I know it can be discussed if it is a type and therefore a capitalized. It is a kwarg parameter and for a user I think it is more natural to specify a kwarg with a small letter. This is also how it is done in other parts of the code.